src/LearnMe.Core/Services/Calendar/Calendar.cs

```diff
@@ -108,7 +108,6 @@ public async Task<bool> DeleteEventAsync(int id)
             int pageNumber)
         {
             // Step 1 - synchronize Google calendar with DB
-            //await _synchronizer.SynchronizeDatabaseWithCalendarByIdAsync(_externalCalendarService, _repository);
             var eventsSynchronizedCount = await _synchronizer.SynchronizeDatabaseWithCalendarByDateModifiedAsync(

```

Comment:
Ta synchronizacja wygląda spoko. Wiadomo, zawsze można by się do czegoś przyczepić ale moim zdaniem na pierwszy rzut oka wygląda ok. Nie odpalałem apki, nie wiem czy działa

Reply:
Podobają mi się te funkcje, tak trzymać. Warto opisywać takie operatory boolowe bo czasem w kodzie nie wynika, dlaczego w jakimś `if` mamy `(calendarEvent.Start != null || calendarEvent.End != null)`

![image](https://user-images.githubusercontent.com/37336963/99898511-1e52d080-2ca2-11eb-861c-941051d8e90b.png)

Reply:
Muszę przyczepić się do tej linii:

![image](https://user-images.githubusercontent.com/37336963/99898540-45110700-2ca2-11eb-9baf-d5cd0a56ac46.png)

Będzie wam to trudno testować jednostkowo. Może warto by było stworzyć interfejs, np `IDate` albo `IClock` w którym byście mieli property jak `Now`?

```csgarp
public interface IDate
{
    DateTime Now { get; }
}
```

Przy testach jednostkowych łatwo byście mogli podmieniać te date i ustawiać dowolnie.

```csharp
 IEnumerable<Event> eventsFromExternalCalendarToBeSynchronized = 
                await externalCalendarService.GetEventsByLastUpdateAsync(lastSynchronizationDateTime, true);
            var updateDateTime = _clock.Now;

// i potem w teście....

_clockMock.SetupGet(x => x.Now).Returns(wasza dowolna data);
```
````

Reply:
Magic stringi jak `cancelled` do jakiegoś pliku resources albo statycznej klasy z constami, już ci o tym pisałem:

![image](https://user-images.githubusercontent.com/37336963/99898578-b9e44100-2ca2-11eb-999d-8ff09e97d2a3.png)

Reply:
Dzięki za radę z IDate 👍 bardzo przydatna.
Magic string.. postaram się na przyszłość wcześniej to refaktorować.
Fajnie że synchronizacja wygląda ok :) - dzięki za feedback